### ISSUE TITLE
About the Tmall datasets

### ISSUE THREAD

**Issue by xiaxin1998** [CCIIPLab/GCE-GNN#3]
Hi,
Thanks for your sharing of this paper!
About the datasets Tmall, I find the website you provide in your paper and download the datasets. But I find that in the test dataset, there are no given labels for it. Only the training set has labels. So would you mind share the Tmall test dataset with sessions' labels for us?
Thanks!

**Comment by Mikrokosmos1997** [comment]
Thank you for your interest in our paper.
The label information in the raw data indicates whether the user is a repeat buyer, which is not related to our task and has not been used in our work. For the Tmall dataset used in our work, the label of each session is the last item in the session.

**Comment by xiaxin1998** [comment]
Thanks four your response!
But when I preprocess the datasets, the statistics are different with your papers.
For Tmall datasets, I use the train_format2.csv and test_format2.csv in the website.
For the preprocess, I firstly get all the sessions, and leave the first 120000 sessions.
For the 120000 sessions, I filter sessions whose lengths are 1 and larger than 40. Then I filter the items who appear less than 5 times. And then split the sessions. 
But I still have different datasets with yours showed in your paper, Table1.
So would you mind share the preprocessed datasets for us or tell me my mistakes in the preprocess?

**Comment by xiaxin1998** [comment]
If you share the preprocessed datasets for us, we'll appreciate. It is more convenient for us to make comparisons with your model.

**Comment by xiaxin1998** [comment]
UPDATE: in the train_format2.csv, there are only 4995 items, but in your paper, there should be 40728 items.

**Comment by Mikrokosmos1997** [comment]
Thank you for your interest.
We have updated the preprocessed Tmall datasets. You can also download the raw Tmall data from the [dropbox link](https://www.dropbox.com/sh/dbzmtq4zhzbj5o9/AAAMMlmNKL-wAAYK8QWyL9MEa/Datasets?dl=0&subfolder_nav_tracking=1) in the paper _Evaluation of Session-based Recommendation Algorithms_.

**Comment by xiaxin1998** [comment]
Thank you very much!!!!!!!!!!
